chart/develop/values.yaml

```diff
@@ -0,0 +1,7 @@
+secretName: svc-questionnaires-secret
+gitlabAuthSecretName: gitlab-auth
+dockerAuthSecretName: docker-auth
+tag: latest
+namespace: usc-chan
+commitSha: some-commit-sha
+appsignalAppEnv: staging
```

Comment:
We have this in svc-auth as well. Do you know why? Because I don't.. You can target separate versions of the helm chart using branches/refnames/commit tags already.

Reply:
In fact we do use refnames already, so this adds a lot of confusion because you can configure it to master-develop or develop-master.

Reply:
No I think this is the environment for appsignal. So when you have a project on app signal it marks it as either staging or production. To differentiate between the two. It can also use the branch name, but I don't know if this is available here?